include/spirv/unified1/spirv.core.grammar.json

```diff
@@ -6374,6 +6374,411 @@
+        { "kind" : "IdRef", "name" : "Direction"},
+        { "kind" : "IdRef", "name" : "TMax"},
+        { "kind" : "IdRef", "name" : "Payload"},
+        { "kind" : "IdRef", "quantifier" : "?", "name" : "Hint" },
+        { "kind" : "IdRef", "quantifier" : "?", "name" : "Bits" }
+      ],
+      "capabilities" : [ "ShaderInvocationReorderEXT" ],
+      "version" : "None"
+    },
+    {
+      "opname" : "OpHitObjectTraceMotionReorderExecuteEXT",
+      "class"  : "Reserved",
+      "opcode" : 5312,
+      "operands" : [
+        { "kind" : "IdRef", "name" : "Hit Object" },
+        { "kind" : "IdRef", "name" : "Acceleration Structure"},
+        { "kind" : "IdRef", "name" : "RayFlags"},
+        { "kind" : "IdRef", "name" : "Cullmask"},
+        { "kind" : "IdRef", "name" : "SBT Record Offset"},
+        { "kind" : "IdRef", "name" : "SBT Record Stride"},
+        { "kind" : "IdRef", "name" : "Miss Index"},
+        { "kind" : "IdRef", "name" : "Origin"},
+        { "kind" : "IdRef", "name" : "TMin"},
+        { "kind" : "IdRef", "name" : "Direction"},
+        { "kind" : "IdRef", "name" : "TMax"},
+        { "kind" : "IdRef", "name" : "Time"},
+        { "kind" : "IdRef", "name" : "Payload"},
+        { "kind" : "IdRef", "quantifier" : "?", "name" : "Hint" },
+        { "kind" : "IdRef", "quantifier" : "?", "name" : "Bits" }
+      ],
+      "capabilities" : [ "ShaderInvocationReorderEXT", "RayTracingMotionBlurNV" ],
+      "version" : "None"
+    },
+    {
+      "opname" : "OpTypeHitObjectEXT",
+      "class"  : "Type-Declaration",
+      "opcode" : 5313,
+      "operands" : [
+        { "kind" : "IdResult" }
+      ],
+      "capabilities" : [ "ShaderInvocationReorderEXT" ],
+      "version" : "None"
+    },
+    {
+      "opname" : "OpReorderThreadWithHintEXT",
+      "class"  : "Reserved",
+      "opcode" : 5314,
+      "operands" : [
+        { "kind" : "IdRef", "name" : "Hint" },
+        { "kind" : "IdRef", "name" : "Bits" }
+      ],
+      "capabilities" : [ "ShaderInvocationReorderEXT" ],
+      "version" : "None"
+    },
+    {
+      "opname" : "OpReorderThreadWithHitObjectEXT",
+      "class"  : "Reserved",
+      "opcode" : 5315,
+      "operands" : [
+        { "kind" : "IdRef", "name" : "Hit Object" },
+        { "kind" : "IdRef", "quantifier" : "?", "name" : "Hint" },
+        { "kind" : "IdRef", "quantifier" : "?", "name" : "Bits" }
+      ],
+      "capabilities" : [ "ShaderInvocationReorderEXT" ],
+      "version" : "None"
+    },
+    {
+      "opname" : "OpHitObjectTraceRayEXT",
+      "class"  : "Reserved",
+      "opcode" : 5316,
+      "operands" : [
+        { "kind" : "IdRef", "name" : "Hit Object" },
+        { "kind" : "IdRef", "name" : "Acceleration Structure"},
+        { "kind" : "IdRef", "name" : "RayFlags"},
+        { "kind" : "IdRef", "name" : "Cullmask"},
+        { "kind" : "IdRef", "name" : "SBT Record Offset"},
+        { "kind" : "IdRef", "name" : "SBT Record Stride"},
```

Comment:
The extension spec names do not include Record for these. One or the other should be changed.